src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 import { useIntl } from 'react-intl';
 import HeaderSection from '../shared/HeaderSection/HeaderSection';
 import { InjectAppServices } from '../../services/pure-di';
 import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
 import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
+import { Switch } from '../shared/Switch/Switch';
 
 export const ContactPolicy = InjectAppServices(() => {
```

Comment:
I don't see you are using this `InjectAppServices`
```suggestion
export const ContactPolicy = ()=> {
```